lib/ui/views/groups/group_details_view.dart

```diff
@@ -128,6 +129,7 @@ class _GroupDetailsViewState extends State<GroupDetailsView> {
             _model.errorMessageFor(_model.ADD_GROUP_MEMBERS));
       }
     }
+    _emails = null;
```

Comment:
Not sure if this `string` datatype is the best way. I think `bool` datatype will satisfy the exact needs.

Reply:
Ok sure! I'll use it.